MetabolicAtlas/MetabolicAtlas
# feat: update doc and api

**Related issue(s) and PR(s)**  
This PR closes #1065. It was co-created by @inghylt and @MalinAhlberg 



**Type of change**  

 
**List of changes made**  

- Add examples on how to use the API for gotEnzymes queries in swagger
- Add information about and link to  gotEnzymes in Documentation 

**Screenshot of the fix**  


**Testing**  

- Instructions on how to test
- Visit `/api/v2/`, make sure everything is working and looks good
- Visit `/documentation#api` and make sure everything looks good

**Further comments**  


**Definition of Done checklist**
